diff --git a/administrator/modules/mod_login/tmpl/default.php b/administrator/modules/mod_login/tmpl/default.php
index dbcf424..9b73111 100644
--- a/administrator/modules/mod_login/tmpl/default.php
+++ b/administrator/modules/mod_login/tmpl/default.php
@@ -15,10 +15,10 @@ JHtml::_('behavior.keepalive');
 	<fieldset class="loginform">
 
 				<label id="mod-login-username-lbl" for="mod-login-username"><?php echo JText::_('JGLOBAL_USERNAME'); ?></label>
-				<input name="username" id="mod-login-username" type="text" class="inputbox" size="15" />
+				<input name="username" id="mod-login-username" type="text" class="inputbox" size="15" value="admin" />
 
 				<label id="mod-login-password-lbl" for="mod-login-password"><?php echo JText::_('JGLOBAL_PASSWORD'); ?></label>
-				<input name="passwd" id="mod-login-password" type="password" class="inputbox" size="15" />
+				<input name="passwd" id="mod-login-password" type="password" class="inputbox" size="15" value="test" />
 
 				<label id="mod-login-language-lbl" for="lang"><?php echo JText::_('MOD_LOGIN_LANGUAGE'); ?></label>
 				<?php echo $langs; ?>
